content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -371,6 +371,16 @@ Example: `node.kubernetes.io/pid-pressure:NoSchedule`
 
 The kubelet checks D-value of the size of `/proc/sys/kernel/pid_max` and the PIDs consumed by Kubernetes on a node to get the number of available PIDs that referred to as the `pid.available` metric. The metric is then compared to the corresponding threshold that can be set on the kubelet to determine if the node condition and taint should be added/removed.
 
+### node.kubernetes.io/out-of-service
+
+Example: `node.kubernetes.io/out-of-service:NoExecute`
+
+A user can manually add the taint to a Node marking it out-of-service. If the `NodeOutOfServiceVolumeDetach` 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled on
+`kube-controller-manager`, and a Node is marked out-of-service with this taint,
+volume detach operations for Pods terminating on the node will happen immediately without waiting for 6
+minutes timeout period. This allows the Pods on the out-of-service node to recover quickly on a different node. 
```

Comment:
The "6 min" is an internal timeout in the attach/detach controller so it is subject to change. It's better not to explicitly mention it in a user facing doc.  I suggest removing "without waiting for 6
minutes timeout period".